public/js/addTask.js

```diff
@@ -1,4 +1,4 @@
-async function taskFormHandler(event) {
+async function newTaskFormHandler(event) {
     event.preventDefault();
   
     const name = document.querySelector('textarea[name="task-body"]').value.trim();
```

Comment:
Each of these form items do have an id associated with them so you could reference them with 
document.querySelector('#whatever').value.trim();
I know for tasks we don't know what whatever is yet!

Reply:
I will wait for Daniels update and update these tags once we coordinate.